scripts/build.py

```diff
@@ -662,7 +662,7 @@ def build_using_webpack(config_path):
 
     python_utils.PRINT('Building webpack')
 
-    cmd = '%s %s --config %s' % (
+    cmd = '%s --max-old-space-size=2400 %s --config %s' % (
```

Comment:
Why do we need this change?

Reply:
> Why do we need this change?

@nithusha21 Because node is out of memory(tested both locally and CI). So I fixed it by specifying limit as suggested by @ashutoshc8101 .

![Screenshot from 2021-04-09 21-57-20](https://user-images.githubusercontent.com/52493334/114262311-ece26380-99fc-11eb-9a67-723418ba3b64.png)

Reply:
limit is suggested to me by @srijanreddy98